recipes/opensees/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "OpenSees" %}
+{% set version = "3.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OpenSees/OpenSees/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: fa805d2db131939b06042568f56b875c40d80a46c96b0d60dbfbfa0fbbceaaa6
+
+build:
+  skip: True  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ stdlib('c') }}
+    - cmake
+    - make
+    - openmpi
+  host:
+    - python 3.11
+    - pip
+    - mumps
+    - eigen 3.4
+    - hdf5 1.14
+    - openmpi
+    - scalapack
+    - tk 8.6
+    - mysql
+    - zlib
+    - mkl
+    - mkl-include
+    - mkl-devel
+  run:
+    - metis
+    - hdf5
+    - libscotch
+    - libaec
+    - libcurl
+    - openssl
+    - scalapack
```

Comment:
```suggestion
```
All these should be removed as they come in via run_exports

Reply:
I could remove hdf5 but I had to leave the other ones, otherwise I get errors for missing runtime dependencies or overlinking when the binaries are analyzed... I do not really understand why though. Would you maybe have any hint to share ?

Reply:
You should add them to `host` instead; then they can be omitted from `run`

Reply:
It seems to work (thanks for the tip!) for anything but scalapack, would that do it in your opinion ? Thanks once again!